app/presenters/local_authority_external_content_presenter.rb

```diff
@@ -0,0 +1,20 @@
+class LocalAuthorityExternalContentPresenter
+  def initialize(local_authority)
+    @local_authority = local_authority
+  end
+
+  def present_for_publishing_api
+    {
+      description: "Website of #{@local_authority.name}",
+      details: {
+        hidden_search_terms: [@local_authority.name],
```

Comment:
I want to say we can probably get rid of `hidden_search_terms` for the purposes of Search v2, given that it's exactly the same as the title anyway 🤔

Reply:
Done